examples/speed_tester/speed_tester.c

```diff
@@ -334,6 +342,10 @@ nf_setup(struct onvm_nf_local_ctx *nf_local_ctx) {
                         if (pkt == NULL)
                                 break;
 
+                        /* Length of the packet cannot exceed preallocated storage size */
+                        if (header.caplen > max_elt_size)
```

Comment:
On line 341: `pkt = rte_pktmbuf_alloc(pktmbuf_pool);` But you continue here without freeing that packet. Perhaps move this continue statement before the allocation occurs?